src/Apps/Artwork/Components/ArtworkSidebar/__tests__/ArtworkSidebarBidAction.test.tsx

```diff
@@ -31,155 +36,578 @@ describe("ArtworkSidebarBidAction", () => {
           artwork(id: "auction_artwork") {
             ...ArtworkSidebarBidAction_artwork
           }
+          me {
+            ...ArtworkSidebarBidAction_me
+          }
         }
       `,
       mockData: {
-        artwork: response,
+        artwork: response.artwork,
+        me: response.me,
       } as ArtworkSidebarBidAction_Test_QueryRawResponse,
     })
   }
 
   describe("for closed auction", () => {
     it("does not display anything", async () => {
-      const wrapper = await getWrapper(ArtworkFromClosedAuction)
+      const wrapper = await getWrapper({
+        artwork: ArtworkFromClosedAuction,
+        me: NotIDVedUser,
+      })
 
       expect(wrapper.html()).toBe(null)
     })
   })
 
   describe("for auction preview", () => {
     it("and not registered bidder", async () => {
-      const wrapper = await getWrapper(ArtworkFromAuctionPreview)
+      const wrapper = await getWrapper({
+        artwork: ArtworkFromAuctionPreview,
+        me: NotIDVedUser,
+      })
 
       expect(wrapper.text()).toContain("Register to bid")
     })
 
     it("with bidder registration pending approval", async () => {
-      const artwork = merge(
-        {},
-        ArtworkFromAuctionPreview,
-        BidderPendingApproval
-      )
-      const wrapper = await getWrapper(artwork)
+      const artwork = merge(ArtworkFromAuctionPreview, BidderPendingApproval)
+      const wrapper = await getWrapper({
+        artwork,
+        me: NotIDVedUser,
+      })
 
       expect(wrapper.text()).toContain("Registration pending")
     })
 
     it("with registered bidder", async () => {
       const artwork = merge({}, ArtworkFromAuctionPreview, RegisteredBidder)
-      const wrapper = await getWrapper(artwork)
+      const wrapper = await getWrapper({ artwork, me: NotIDVedUser })
 
       expect(wrapper.text()).toContain("Registration complete")
     })
-  })
 
-  describe("for open auction", () => {
-    it("with open registration and not registered bidder ", async () => {
-      const artwork = merge(
-        {},
-        ArtworkFromTimedAuctionRegistrationOpen,
-        NotRegisteredToBid
-      )
-      const wrapper = await getWrapper(artwork)
-
-      expect(wrapper.text()).toContain("Place max bid")
-      expect(wrapper.text()).toContain("$900")
-      expect(wrapper.text()).toContain("Bid")
-    })
+    describe("when the sale requires identity verification", () => {
+      const SaleWithArtwork = merge(ArtworkFromAuctionPreview, {
```

Comment:
a similiar approach led to flakey tests, so I should just inline this into the relevant specs